javaScriptFunctions_arrowFunctions/codeChallenge/random.js

```diff
@@ -0,0 +1,11 @@
+function generateRandomNumber (high, low) {
+  if(high > low) {
+    const randomNum = Math.floor(Math.random() * (high - low + 1)) + low;
+    return randomNum;
+  } else {
+    return 'Invalid Input!';
+  }
+};
+
+console.log(generateRandomNumber(10, 7));
+console.log(generateRandomNumber(300, 1));
```

Comment:
Same here. No newline at EOF, but there's no need to update here. Let's just be mindful of this next time.